### PR TITLE
Setup CI of ESlint with Prettier 

### LINKED ISSUE #6: Setup CI of ESlint with prettier
https://github.com/LiorKGOW/My-Guitar-Store/pull/13

### PR DESCRIPTION
**Pull request by archanaserver**
This PR contains the following changes:

- Installed and configured ESlint with Prettier for the project. Run command- `npx prettier --write app/` to format the `app` directpry and `npx prettier --check .` in CI to make sure that your project stays formatted.
- Added CI with ESlint
- Downgrade to use React 16.14.0, same as Foreman ([Reference](https://github.com/LiorKGOW/My-Guitar-Store/pull/13))

Resolves #6 Installed and configured CI of ESlint with Prettier

### CODE REVIEW

**Comment by archanaserver** [comment]
Build succeeded: https://github.com/girijaasoni/newsletterApplicaton/actions/runs/3590063073/jobs/6043106505 🚀

**Review by girijaasoni — APPROVED** [review]

File: .github/workflows/eslint.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+name: ESLint
+on: [push, pull_request]
```
Comment (Ron-Lavi) [inline .github/workflows/eslint.yml]: Can you run it only on `pull_request`? Otherwise we will see two GitHub actions running for ESlint on each PR.

**Review by Ron-Lavi — COMMENTED** [review]
Thanks @archanaserver !

File: .github/workflows/eslint.yml (hunk 1 above)
Comment (archanaserver) [reply .github/workflows/eslint.yml]: @Ron-Lavi updated the changes: https://github.com/girijaasoni/newsletterApplicaton/pull/18/commits/75fae33b684986d689a7f92c00a35e90c5485210

**Comment by Ron-Lavi** [comment]
Thanks @archanaserver @girijaasoni !

**Pull request merged**